MapPointMapper/ViewController.swift

```diff
@@ -108,14 +109,31 @@ class ViewController: NSViewController, MKMapViewDelegate, NSTextFieldDelegate {
     renderer.strokeColor = colorWell.color
     return renderer
   }
-  
-  // MARK: NSTextFieldDelegate
-  override func keyUp(theEvent: NSEvent) {
```

Comment:
:+1: i think that should work. If it misses something we can always fix it later